autoload/db_ui/schemas.vim

```diff
@@ -34,9 +34,16 @@ let s:postgres_list_schema_query = "
     \   and pg_catalog.has_schema_privilege(current_user, nspname, 'USAGE')
     \ order by nspname"
 
-let s:postgres_tables_and_views = "
-      \ SELECT table_schema, table_name FROM information_schema.tables UNION ALL
-      \ select schemaname, matviewname from pg_matviews;"
+if !empty(g:db_ui_use_redshift)
```

Comment:
Use `get` to allow it to be undefined.
```suggestion
if !empty(get(g:, 'db_ui_use_redshift', 0))
```

Reply:
Made all requested changes, including changing the name to `g:db_ui_use_postgres_views` and defaulting it to 1.

Using postgres views:
![dadbod_using_postgres_views](https://github.com/kristijanhusak/vim-dadbod-ui/assets/59935328/e76cd215-9893-4641-a47c-ae438d0729c0)

Overwriting postgres views:
![dadbod_overwriting_default_postgres_views](https://github.com/kristijanhusak/vim-dadbod-ui/assets/59935328/000c07dc-08a8-46cf-96f0-0ec1014889c3)